lab-raziyeh/app/component/signup/signup.html

```diff
@@ -0,0 +1,48 @@
+<!--<div class="signup">
+  <form
+    class=""
+    ng-submit="signupCtrl.signup(signupCtrl.user)"
+    novalidate>
+
+    <input class="form-control" 
+    ng-minlength="6" required
+    type="text" placeholder="Username" ng-model="signupCtrl.user.username">
+
+    <input class="form-control" 
+    required type="email" placeholder="Email" ng-model="signupCtrl.user.email">
+
+    <input class="form-control"
+    required type="password" ng-minlength="8"
+    placeholder="Password" ng-model="signupCtrl.user.password">
+    
+    <input type="submit" class="btn btn-info" value="Sign Up">
+  </form>
+</div>-->
```

Comment:
@Raziyehbazargan best to pull out large comment blocks from the `master` branch - to keep things production ready